src/openagi/agent.py

```diff
@@ -164,6 +164,9 @@ def get_previous_task_contexts(self, task_lists: TaskLists):
         if task_summaries:
             return "\n".join(task_summaries).strip()
         return "None"
+    
+    def _get_planner_by_id(self,planner_id:str):
+        pass
```

Comment:
planner id: for the continuous task creation. This needs to be brainstormed further to assign the task to worker